### PR TITLE
update 3.x readme

### PR DESCRIPTION
**Pull request by RangiLyu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -338,6 +349,7 @@ MMDetection 是一款由来自不同高校和企业的研发人员共同参与
 
 ## OpenMMLab 的其他项目
 
+- [MMEngine](https://github.com/open-mmlab/mmengine): OpenMMLab 深度学习模型训练库。
```
Comment (zhouzaida) [inline README_zh-CN.md:352]:
```suggestion
- [MMEngine](https://github.com/open-mmlab/mmengine): OpenMMLab 深度学习模型训练库
```

File: README.md [hunk 2]
```diff
@@ -71,44 +71,50 @@ The master branch works with **PyTorch 1.5+**.
 
 </details>
 
-Apart from MMDetection, we also released a library [mmcv](https://github.com/open-mmlab/mmcv) for computer vision research, which is heavily depended on by this toolbox.
+Apart from MMDetection, we also released [MMCV](https://github.com/open-mmlab/mmcv) for computer vision research and [MMEngine](https://github.com/open-mmlab/mmengine) for model training, that are heavily depended on by this toolbox.
```
Comment (ZwwWayne) [inline README.md:74]: that are -> which are
list MMEngine before MMCV

File: README.md [hunk 3]
```diff
@@ -71,44 +71,50 @@ The master branch works with **PyTorch 1.5+**.
...
 
-- Support dedicated `MMDetWandbHook` hook
-- Support [ConvNeXt](configs/convnext), [DDOD](configs/ddod), [SOLOv2](configs/solov2)
-- Support [Mask2Former](configs/mask2former) for instance segmentation
-- Rename [config files of Mask2Former](configs/mask2former)
-
-Please refer to [changelog.md](docs/en/changelog.md) for details and release history.
-
-For compatibility changes between different versions of MMDetection, please refer to [compatibility.md](docs/en/compatibility.md).
+- Faster and smoother training based on [MMEngine](https://github.com/open-mmlab/mmengine).
```
Comment (ZwwWayne) [inline README.md:80]: - Unifies interfaces of all components based on MMEngine.
- Faster training and testing speed with complete support of mixed precision training.
- Provides more strong baselines and a general semi-supervised object detection framework. See [tutorial of semi-supervised detection](https://mmdetection.readthedocs.io/en/v3.0.0rc0/user_guides/semi_det.html).
- Allows any kind of single-stage model as an RPN in a two-stage model. See [tutorial](https://mmdetection.readthedocs.io/en/v3.0.0rc0/user_guides/single_stage_as_rpn.html).

File: README.md [hunk 4]
```diff
@@ -71,44 +71,50 @@ The master branch works with **PyTorch 1.5+**.
...
-- [export a model to ONNX](docs/en/tutorials/pytorch2onnx.md)
-- [export ONNX to TRT](docs/en/tutorials/onnx2tensorrt.md)
-- [weight initialization](docs/en/tutorials/init_cfg.md)
-- [how to xxx](docs/en/tutorials/how_to.md)
-- [semi-supervised_detection](docs/en/tutorials/semi_det.md)
+Please see [Overview](https://mmdetection.readthedocs.io/en/3.x/get_started.html) for the general introduction of MMDetection.
+
+For detailed user guides and advanced guides, please refer to our [documentation](https://mmdetection.readthedocs.io/en/3.x/):
+
+- User Guides
```
Comment (ZwwWayne) [inline README.md:95]: Consider using `<detaile>` to squeeze some details

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**